src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="searchForm searchFormBig"
```

Comment:
general convention for naming classes in css with use `kebab-case` or some special covention as for BEM metodology
```suggestion
    <form class="search-form search-form-big"
```

Reply:
another 2 tests passed, but 4 didn`t